### PR TITLE
HAAR-1276:  Move Delius Mappings from Auth to Manage users api

### PR DESCRIPTION
**Pull request by ravmoj**
Moving Delius Mappings from Auth to Manage users api

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/delius/UserApiService.kt [hunk 1]
```diff
@@ -4,25 +4,52 @@ import org.slf4j.LoggerFactory
...
+      serviceWebClientUtils.getIgnoreError(
+        "/users/$username/details",
+        DeliusUser::class.java,
+      )
+
+    if (users != null) {
+      users = mapUserDetailsToDeliusUser(users)
+    }
+    return users
   }
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/delius/UserApiService.kt:39]: can tidy this up with something similar to:
```
  return serviceWebClientUtils.getIgnoreError
      "/users/$username/details",
      DeliusUser::class.java,
    )?.apply{ mapUserDetailsToDeliusUser(users) }
```

**Review by karenmillermoj — DISMISSED** [review]
lgtm - can improve one bit of code slightly (not tested tho!) - see comment

File: src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/delius/UserApiService.kt (hunk 1 above)
Comment (ravmoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/adapter/delius/UserApiService.kt]: Updated

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**